index.html

```diff
@@ -0,0 +1,130 @@
+      <ul class="menu">
+        <li><a class="nav-link" href="#about-me"> <img src="assets/images/facebook.png" alt=""> </a></li>
+        <li><a class="nav-link" href="#about-me"> <img src="assets/images/twitter.png" alt=""> </a></li>
+        <li><a class="nav-link" href="#contact-form">English</a></li>
+      </ul>
+    </div>
+    <div id="bottom-bar">
+      <div id="first-item">
+        <span id="logo-placeholder"><a class="nav-link" href="index.html#intro"><img src="assets/images/Lome_Fintech_LogoB_59.png" alt=""></a></span>
+        <nav id="nav-bar">
+          <ul class="menu">
+            <li><a class="nav-link" href="about.html#about">About</a></li>
+            <li><a class="nav-link" href="index.html#main-program">Program</a></li>
+            <li><a class="nav-link" href="index.html">Join</a></li>
+            <li><a class="nav-link" href="index.html#partner">Sponsor</a></li>
+            <li><a class="nav-link" href="index.html#news">News</a></li>
+            <li><a class="nav-link" href="index.html#partner"><button class="btn">The Challenge</button></a></li>
+          </ul>
+        </nav>
+      </div>
+      <!-- Hamburger icon -->
+      <label id="second-item" class='menu-button-container' for="menu-button-container">
+        <img id="cancel-icon" src="assets/images/close_mobile_menu.png" alt="Cancel Menu">
+        <img id="menu-icon" src="assets/images/mobile_menu.png" alt="Cancel Menu">
+      </label>
+    </div>
+  </header>
+  <!-- Main content -->
+  <main class="content">
+    <section id="intro">
+      <!--       <h1 id="hello">"Hello. Sharing world"</h1>
+      <p id="title">
+        CONFERENCE INTERNATIONALE DES EXPERTS REGIONAUX EN CYBERSECURITE
+      </p> -->
+      <img src="assets/images/intro2.png" alt="">
+      <p id="description">
+        The Challenge is open to all Fintechs on the African continent. Interested Fintechs can apply by sending details of their product as well as a demo  <a href="http://" target="_blank" rel="noopener noreferrer">here</a>. These products must address one or more of a specific set of pain points which can be found <a href="http://" target="_blank" rel="noopener noreferrer">here</a>.
+      </p>
+      <p id="when">
+        2022.11.09(WED) ~ 12(SAT)
+      </p>
+      <p id="where">
+        Salle Evala, Hotel 2 Fevrier, Lomé - TOGO
+      </p>
+    </section>
+    <section id="main-program">
+      <h1>Main Program</h1>
+      <span class="program-bar"></span>
+      <ul class="program">
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Lecture</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Workshops</p>
+          <p id="description">
```

Comment:
It will be good if you make the design of the program section match our guideline design as closely as possible. By implementing them, I believe you'll have one or two things to learn and add to your skills arsenal.

Note that:

- The section has a pattern image background
- The individual cards are a bit transparent

Your design:
![image](https://user-images.githubusercontent.com/20554583/198849125-6abf3f69-0e88-4e95-8b75-db88c1deb695.png)


Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/198849136-cf1b3099-b22c-4d67-aa6b-dec74757492d.png)